serverlessworkflow/modules/ROOT/pages/cloud/operator/referencing-resource-files.adoc

```diff
@@ -0,0 +1,155 @@
+= Referencing Resource Files in the Workflow
```

Comment:
Is "resource files" a description people will understand?  Not sure about a better name though
```suggestion
= Referencing Additional Files from the Workflow
```